<?xml version="1.0"?>
<launch>
    <node name="relocalization" pkg="relocalization" type="relocalization" output="screen" >
        <param name="pcd_path" value="/home/tjurm/navi_ws/Sentry25NAVI/src/Point-LIO-master/PCD/505_1.pcd"/>
        <param name="obs_path" value="/home/tjurm/navi_ws/Sentry25NAVI/src/Point-LIO-master/PCD/505obs_1.pcd"/>
        <param name="leaf_size" value="0.2"/>
        <param name="map_leaf_size" value="0.2"/>
        <param name="max_dist_sq" value="2.0"/>
        <param name="num_threads" value="8"/>
        <param name="max_iterations" value="20"/>
        <param name="num_neighbors" value="2.0"/>
        <param name="diverge_threshold" value="120.0"/>  <!-- 40 -->
        <param name="precise_diverge_threshold" value="120.0"/>
        <param name="debug_en" value="true"/>
        <param name="freq" value="5"/>
        <param name="max_frame" value="1"/>
        <param name="use_stl_cloud" type="bool" value="false"/>
        <param name="lidar_height" type="double" value="0.32"/>
        <param name="lidar_pitch" type="double" value="0.349"/>
    </node>
</launch>

<!-- launch-prefix="xterm -e gdb -ex run a-r-gs " -->
